src/effects/RealtimeEffectList.cpp

```diff
@@ -90,7 +90,8 @@ RealtimeEffectList::AddState(std::shared_ptr<RealtimeEffectState> pState)
       Publisher<RealtimeEffectListMessage>::Publish({
          RealtimeEffectListMessage::Type::Insert,
          mStates.size() - 1,
-         { }
+         { },
+         pState
```

Comment:
I should have noticed this last round -- `pState` is moved-from therefore null.  Nothing in non-modality depends on this but the events should anyway behave as advertised.

Reply:
Ironically, MSVC now has a diagnostic for that
![image](https://user-images.githubusercontent.com/2660628/184398360-3106dde9-aff2-4904-9b65-2bfe93e18d75.png)